opm/input/eclipse/Schedule/Well/WellPropertiesKeywordHandlers.cpp

```diff
@@ -145,10 +145,11 @@ void handleWEFAC(HandlerContext& handlerContext)
         const auto well_names = handlerContext.wellNames(wellNamePattern);
 
         const double& efficiencyFactor = record.getItem("EFFICIENCY_FACTOR").get<double>(0);
+        const bool useEfficiencyInExtNetwork = DeckItem::to_bool(record.getItem("EXTENDED_NETWORK_OPT").getTrimmedString(0));
```

Comment:
This item should only matters for extended network model.  Currently we do not have information about whether the network is standard network or extended network, we should introduce it to do differently between standard network and extended network. 

The same applies to `GEFAC` processing.

Reply:
Good point, but since we actually can support this for both standard and extended networks, maybe we should. Will have a go and update names/descriptions accordingly.